File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    
    str_even = ""
    str_odd = ""
    
    for i in range(len(num_list)):
        if num_list[i] % 2 == 0:
            str_even += str(num_list[i])
        else:
            str_odd += str(num_list[i])
    
    answer = int(str_even) + int(str_odd)
    
    return answer

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        
        String even = "";
        String odd = "";
        
        for(int i=0; i<num_list.length; i++){
            if(num_list[i] % 2 == 0){
                String even_str = Integer.toString(num_list[i]);
                even += even_str;
            }
            else {
                String odd_str = Integer.toString(num_list[i]);
                odd += odd_str;
            }
        }
        
        int even_int = Integer.parseInt(even);
        int odd_int = Integer.parseInt(odd);
        
        answer = even_int + odd_int;
        
        return answer;
    }
}